tests/hpc/munge_slave.pm

```diff
@@ -11,7 +11,7 @@
 # of this package
 # Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>, soulofdestiny <mgriessmeier@suse.com>
 
-use base "hpcbase";
+use base "opensusebasetest";
```

Comment:
There's a `$self->enable_and_start` in line 38.

Reply:
Changed!